### ISSUE TITLE
cargo install wasm-bindgen-cli fails on windows (travis)

### ISSUE THREAD

**Issue by najamelan** [rustwasm/wasm-bindgen#2006]
### Describe the Bug

It fails to compile with linker errors seemingly related to libcurl. Happens on both stable and nightly. I runs fine in linux and mac os.

### Steps to Reproduce

1. Go to https://travis-ci.org/najamelan/async_executors/jobs/652183842
2. The travis.yml: https://travis-ci.org/najamelan/async_executors/jobs/652183842/config

I haven't found time to debug this further, but I thought I'd report it because others might start to see builds failing.

Last part of the log:
```
 = note: C:\Users\travis\AppData\Local\Temp\cargo-install2hNVHf\release\deps\libcurl_sys-c776d319576f8ecc.rlib(url.o):url.c:(.text$Curl_init_userdefined+0x9): undefined reference to `__imp___acrt_iob_func'

          C:\Users\travis\AppData\Local\Temp\cargo-install2hNVHf\release\deps\libcurl_sys-c776d319576f8ecc.rlib(cookie.o):cookie.c:(.text$Curl_cookie_init+0x27b): undefined reference to `__imp___acrt_iob_func'

          C:\Users\travis\AppData\Local\Temp\cargo-install2hNVHf\release\deps\libcurl_sys-c776d319576f8ecc.rlib(cookie.o):cookie.c:(.text$Curl_flush_cookies+0x17f): undefined reference to `__imp___acrt_iob_func'

          C:\Users\travis\AppData\Local\Temp\cargo-install2hNVHf\release\deps\libcurl_sys-c776d319576f8ecc.rlib(formdata.o):formdata.c:(.text$Curl_getformdata.part.0+0x266): undefined reference to `__imp___acrt_iob_func'

          C:\Users\travis\AppData\Local\Temp\cargo-install2hNVHf\release\deps\libcurl_sys-c776d319576f8ecc.rlib(setopt.o):setopt.c:(.text$Curl_vsetopt+0x3b6c): undefined reference to `__imp___acrt_iob_func'

          C:\Users\travis\AppData\Local\Temp\cargo-install2hNVHf\release\deps\libcurl_sys-c776d319576f8ecc.rlib(mprintf.o):mprintf.c:(.text$curl_mprintf+0x29): more undefined references to `__imp___acrt_iob_func' follow
```

**Comment by alexcrichton** [comment]
Thanks for the report! This is a build issue however with one of the dependencies, specifically `curl-sys`. The fix here should be in [nightly Rust at this point](https://github.com/rust-lang/rust/pull/67429) and otherwise I'd recommend using the MSVC toolchain wherever possible. I don't think is a wasm-bindgen issue, however, so I'm going to close this.

**Comment by najamelan** [comment]
Thanks for clarifying, and good that it's already fixed. I suppose it should go away as soon as travis updates the rust nightly. I'll keep an eye out to see if it solves itself. The issue was detected on:
```
rustc 1.43.0-nightly (0176a9eef 2020-02-17)
$ rustup --version
rustup 1.21.1 (7832b2ebe 2019-12-20)
$ cargo --version
cargo 1.43.0-nightly (3c53211c3 2020-02-07)
```

**Comment by najamelan** [comment]
Still seeing this on:
```
$ rustc --version
rustc 1.43.0-nightly (6d0e58bff 2020-02-23)
$ rustup --version
rustup 1.21.1 (7832b2ebe 2019-12-20)
$ cargo --version
cargo 1.43.0-nightly (e57bd0299 2020-02-21)
```
I doubt the problem was solved in rust-lang/rust#67429.

**Comment by alexcrichton** [comment]
Ah ok, sorry you're still seeing this! In any case it's an issue for the `curl` crate or upstream rust-lang/rust rather than this crate.
